samples/test/create_realm.test.js

```diff
@@ -29,6 +30,11 @@ describe('Game Servers Create Realm Test', () => {
   const client = new RealmsServiceClient();
   let realmId;
 
+  before(async () => {
+    const projectId = await client.getProjectId();
+    cleanup(client, null, projectId, LOCATION);
```

Comment:
Going to guess this is an async function, and we need to `await` it :)

Reply:
Here and below

Reply:
D'oh! You are 100% correct.

Done.